src/igvfd/tests/data/inserts/prediction_set.json

```diff
@@ -9,7 +9,7 @@
         "donors": [
             "igvf:alias_human_donor1"
         ],
-        "file_set_type": "functional effect",
+        "file_set_type": "binding effect",
```

Comment:
can you create a new object in the data insert instead of replacing the `file_set_type` in an existing one.

Reply:
yes will do